lib/forecast.py

```diff
@@ -547,7 +566,9 @@ def prepare(self, n, train_len, steps):
         """
         x_train = np.linspace(0, 1, train_len).reshape(-1, 1)
         pas = x_train[1, 0] - x_train[0, 0]
-        x_pred = np.arange(0, (len(self) + steps) * pas, pas).reshape(-1, 1)
+        # x_pred = np.arange(0, (len(self) + steps) * pas, pas).reshape(-1, 1) #TODO: Check this len(self) + steps logic
```

Comment:
Has this defintely been checked? I recall discussing this - but I can't remember what we decided on. I suspect the old implementation was correct?